File: app/src/main/java/com/sagar/watchnext/network/models/tv/videos/Videos.java

package com.sagar.watchnext.network.models.tv.videos;

import java.util.List;

import com.google.gson.annotations.SerializedName;

public class Videos {

    @SerializedName("id")
    private int id;

    @SerializedName("results")
    private List<Video> videos ;

    public int getId() {
        return id;
    }

    public List<Video> getVideos() {
        return videos;
    }
}

File: app/src/main/java/com/sagar/watchnext/network/models/tv/videos/Videos.kt
package com.sagar.watchnext.network.models.tv.videos

import com.google.gson.annotations.SerializedName

class Videos {

    @SerializedName("id")
    val id: Int = 0

    @SerializedName("results")
    val videos: List<Video>? = ArrayList()
}
